web-registry/public/_redirects

```diff
@@ -1,2 +1,10 @@
-https://registry.regen.network/*  https://app.regen.network/:splat 301!
-/*    /index.html   200
+https://registry.regen.network/_ https://app.regen.network/:splat 301!
+https://app.regen.network/projects/C02-002 https://app.regen.network/project/C02-002 301!
```

Comment:
wouldn't that work to just use relative paths?
we might also want to to add off-chain projects which might have been shared publicly in the past:
/wilmot, /cavan-station, /woodburn and /pillango